src/Client/CurlMultiHandler.php

```diff
@@ -16,7 +16,7 @@
  *
  * @property resource $_mh Internal use only. Lazy loaded multi-handle.
  */
-#[AllowDynamicProperties]
+#[\AllowDynamicProperties]
```

Comment:
Why do we need the backslash?

Reply:
Without the slash, this attribute resolves to `GuzzleHttp\Ring\Client\AllowDynamicProperties` instead of `\AllowDynamicProperties`

Reply:
I didn't know that the Attribute are under namespace. Thanks!

Reply:
Same, I didn't notice until I tried it E2E :laughing: